src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbol.cs

```diff
@@ -16,18 +16,218 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
-    internal sealed class SourceOrdinaryMethodSymbol : SourceOrdinaryMethodSymbolBase
+    internal class SourceOrdinaryMethodSymbol : SourceOrdinaryMethodSymbolBase
```

Comment:
not sure if it would be preferable here to do `ConcreteType derives-from ConcreteType`.  or if you would prefer an abstract base, and then two specific concrete subtypes.  one of them would be entirely empty, except for a forwarding constructor.

Reply:
I would go with `abstract` base. The main cost is having to come up with a type name ;-) 
Not sure what others think.